parm/snow/obs/config/bufr2ioda_mapping.yaml.j2

```diff
@@ -2,3 +2,4 @@ mkdir:
 - '{{ DATA }}/obs'
 copy:
 - ['{{PARMgfs}}/gdas/snow/obs/config/bufr_sfcsno_mapping.yaml', '{{ DATA }}/obs/']
+- ['{{PARMgfs}}/gdas/snow/obs/config/ioda_bufr_python_encoder.py', '{{ DATA }}']
```

Comment:
does the python script need copied? and should it go here? we might need to talk to @emilyhcliu about the progress of SPOC

Reply:
Thanks @CoryMartin-NOAA for your suggestions. Do you agree if I put this python code to `sorc/gdas.cd/parm/snow/`, so that this code can be used directly? 
@emilyhcliu Any suggestions?

Reply:
I moved this py code to the parm/gdas/snow/ directory for a temporary solution. @emilyhcliu Please let me know if you have any other suggestions. Thanks.

Reply:
Scripts shouldn't be in parm/, until we move to SPOC, it should probably go here: https://github.com/NOAA-EMC/GDASApp/tree/develop/ush/ioda/bufr2ioda

Reply:
Understood. Wait for @emilyhcliu suggestions.